reV/generation/generation.py

```diff
@@ -265,6 +287,69 @@ def run(cls, points_control, tech=None, res_file=None, output_request=None,
 
         return out
 
+    def _parse_gid_map(self, gid_map):
```

Comment:
Should this go in GenBase?

Reply:
I don't think so... I didnt really see any reason to have this available in the econ module. There's not a real use case I can come up with and it gets super confusing because there would be res_gids -> gen_gids -> econ_gids.

Reply:
That was my suspicion and works for me